test/fixtures/fixtures.bzl

```diff
@@ -1,25 +1,15 @@
 """Functions for updating test fixtures."""
 
-load("@bazel_skylib//lib:paths.bzl", "paths")
 load(
     "//xcodeproj:xcodeproj.bzl",
+    "XcodeProjOutputInfo",
     "make_xcodeproj_rule",
     "xcodeproj",
-    "XcodeProjOutputInfo",
 )
 
-# Utility
-
-def _install_path(xcodeproj):
-    # "example/ios_app/p.xcodeproj" -> "test/fixtures/ios_app/project.xcodeproj"
-    return paths.join(
-        "test/fixtures",
-        xcodeproj.short_path.split("/")[1],
-        "project.xcodeproj",
-    )
-
 # Transition
 
+# buildifier: disable=unused-variable
```

Comment:
Can we instead use the underscore for this?

Reply:
Sorry. I don't follow. Do you mean the warning category? I am basing them off of [this list](https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md).

Reply:
We can rename the unused variables to have a leading underscore instead of ignoring the warning.

Reply:
I think you meant the variable names. 🙂 That does work. I removed the directive and added the underscores.